backend/gene_info/api/v1.py

```diff
@@ -0,0 +1,33 @@
+import string
+from flask import jsonify, make_response
+from backend.gene_info.api.ncbi_provider import NCBIAPIException, NCBIProvider, NCBIUnexpectedResultException
+
+
+def gene_info(geneID: string):
+    provider = NCBIProvider()
+
+    # search for gene UID from ensembl ID
+    try:
+        uid = provider.fetch_gene_uid(geneID)
+    except NCBIAPIException:
+        return make_response(jsonify("Failed search of NCBI database, API key issue"), 404)
```

Comment:
Create an exception and [raise it]( https://github.com/chanzuckerberg/single-cell-data-portal/blob/ff7ebfb7218c4f26176fde905613bfa0c73d4712/backend/corpora/lambdas/api/v1/dataset.py#L23). This is how the exceptions are [defined](https://github.com/chanzuckerberg/single-cell-data-portal/blob/ff7ebfb7218c4f26176fde905613bfa0c73d4712/backend/corpora/common/utils/http_exceptions.py#L41). You don't need to put your errors here but do inherit the problemException and raise it. This will allow the flask app to handle the error in a uniform way.